examples/simple-dg.py

```diff
@@ -0,0 +1,527 @@
+from __future__ import division, print_function
```

Comment:
Why is this duplicated here? Are there any changes wrt the original?

Reply:
This was just due to merging with the other array context branch I believe. Removed.